airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -540,6 +540,22 @@ def is_active(self) -> bool:
             self.value.lower_bound is not None or self.value.upper_bound is not None
         )
 
+class _IsDagScheduledFilter(BaseParam[bool]):
+    """Filter on timetable_description."""
+
+    def to_orm(self, select: Select) -> Select:
+        if not self.value is None and self.skip_none:
+            if not self.value:
```

Comment:
You have already added `if not self.value is None` above which makes this `if not self.value` always true. We may either need two if separately if they are separate cases or merge them into one could be make sense too

Reply:
Thanks for the details! Indeed, I overlooked the condition, these are different conditions